### PR TITLE
Create Unit Tests for `controllers` and `cors` for Grading Service 

### LINKED ISSUE #45: Create High Coverage Unit Tests for Grading Service

### PR DESCRIPTION
**Pull request by tryyang2001**
# Pull Request

## Description

Create more unit tests for controllers including the `base-controller` and the `grading-controller`, as well as the `get-handler` and `cors`. 

## Related Issue(s)

Closes #45 

## Screenshots (if applicable)

image
In overall, the line coverage is 98.93%, with 89.65% branch coverage, which is more than sufficient for the requirement (I hope)

## Checklist

- [x] I have checked that the changes included in the PR are intended to merge to `master` or any destination branch.
- [x] I have verified that the new changes do not break any existing functionalities, unless the new changes are intended and have approved by the team.
- [x] I will take care of the merging and delete the side-branch after the PR is merged.

### CODE REVIEW

File: backend/grading-service/src/models/its/error-feedback.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
 export type ErrorFeedback = {
-  lineNumber: number;
-  hintStrings: string[];
+  line: number;
+  hints: string[];
 };
```
Comment (WillCWX) [inline backend/grading-service/src/models/its/error-feedback.ts:4]: Ah so the change was done here

**Review by WillCWX — COMMENTED** [review]
Might need to merge the other pull request before reviewing
A lot of changes here to `api-wrapper.test.ts` is found on the other pr and overlaps

File: backend/grading-service/src/models/its/error-feedback.ts (hunk 1 above)
Comment (tryyang2001) [reply backend/grading-service/src/models/its/error-feedback.ts:4]: Paiseh, forgot to push this changes to the previous PR xD

**Review by WillCWX — APPROVED** [review]
LGTM

**Pull request merged**